lib/matplotlib/tests/test_mathtext.py

```diff
@@ -190,6 +191,24 @@ def test_mathtext_rendering(baseline_images, fontset, index, test):
              horizontalalignment='center', verticalalignment='center')
 
 
+cur_lightweight_math_tests = list(
+    filter(lambda x: x[1] is not None, enumerate(lightweight_math_tests)))
+
+
+@pytest.mark.parametrize('index, test', cur_lightweight_math_tests,
+                         ids=[str(idx) for idx, _ in cur_lightweight_math_tests])
+@pytest.mark.parametrize('fontset', ['dejavusans'])
+@pytest.mark.parametrize('baseline_images', ['mathtext1'], indirect=True)
+@image_comparison(baseline_images=None, extensions=['png'])
+def test_mathtext_rendering_lightweight(baseline_images, fontset, index, test):
+    fig = plt.figure(figsize=(5.25, 0.75))
+    fig.text(0.5, 0.5, test, math_fontfamily=fontset,
+             horizontalalignment='center', verticalalignment='center')
+
+
+font_tests = list(filter(lambda x: x[1] is not None, enumerate(font_tests)))
```

Comment:
I think this filtering should stay right below the definition of `font_tests`, because it's only a helper to sanetize the list (namely, to skip deleted tests).

Reply:
I added another commit (actually, a first commit) to factor out test skipping for all of `mathtext_rendering`, `mathtext_rendering_lightweight`, and `mathfont_rendering`.